src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -32,23 +32,95 @@
  */
 class PropertyAccessor implements PropertyAccessorInterface
 {
-    private const VALUE = 0;
```

Comment:
Can you please rebase on the `3.4` branch and change the target branch accordingly to get rid of these unrelated changes?

Reply:
Rebased